systers_portal/templates/meetup/about.html

```diff
@@ -0,0 +1,7 @@
+{% extends "meetup/base.html" %}
```

Comment:
I would suggest doing something similar you had in the mockup - http://prntscr.com/7i10nt

1. Remove the `<hr>` at the bottom.
2. Create a slim border around the about page
3. Increase the font-size of the About paragraph
4. Add some pleasing visual padding and margin